.github/workflows/registry.yml

```diff
@@ -71,14 +71,19 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@master
+      - uses: oprypin/find-latest-tag@v1
+          with:
+            repository: oam-dev/kubevela
+            releases-only: true
+          id: latest_tag
       - name: Get the version
         id: get_version
         run: |
           VERSION=${GITHUB_REF#refs/tags/}
           if [[ ${GITHUB_REF} == "refs/heads/master" ]]; then
             VERSION=latest
           fi
-          echo ::set-output name=VERSION::${VERSION}
+          echo ::set-output name=VERSION::${{ steps.latest_tag.outputs.tag }}-${VERSION}
```

Comment:
revert this line

Reply:
we should add a new Variable for this case. The old version(latest) we still need for image.

Reply:
Fixed.